addPlayer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require_once 'Player.php';
+require_once 'pdo-connection.php';
+require_once 'PlayerManager.php';
+
+// Retrieve the form data
+$name = $_POST['name'];
```

Comment:
You could combine this into one piece of code... Add the $_POST stuff into the new Player declaration

Reply:
Nice to have... unneccesary at this stage but maybe in future